views/unsupported.handlebars

```diff
@@ -8,6 +8,7 @@
     </a>
   {{else}}
     <div class="description" data-l10n-id="notSupportedDetail">Unfortunately this browser does not support the web technology that powers Firefox Send. You’ll need to try another browser. We recommend Firefox!</div>
+    <div class="description"><a href="https://github.com/mozilla/send/blob/master/docs/faq.md#why-is-my-browser-not-supported" data-l10n-id="notSupportedLink" target="_blank" rel="noopener noreferrer"></a></div>
```

Comment:
Absolutely...

Warning, the combination of `<noscript>` on the /unsupported/* route looks terrible (~~mostly~~ _entirely_ due to the fact my noscript hack doesn't seemingly conform to any style guide in the known universe).

<img width="772" alt="firefox_send" src="https://user-images.githubusercontent.com/557895/28938600-67a950c2-7843-11e7-997a-e61f137b4ea3.png">